src/util/tumblr_helpers.js

```diff
@@ -68,8 +68,10 @@ export const createEditRequestBody = postData => {
     interactabilityReblog,
 
     canBeTipped,
-    hasCommunityLabel,
-    communityLabelCategories
+    communityLabels: {
+      hasCommunityLabel,
+      communityLabelCategories
```

Comment:
not seeing `communityLabels.communityLabelCategories` in timeline objects, but am seeing `communityLabels.categories`

```suggestion
      categories: communityLabelCategories
```

Reply:
> what am I even doing this is an embarrassment

mhmm yes good

Reply:
you're fine. remember: a mistake approved is a mistake shared. we're equally responsible